java-api-core/src/com/kinvey/java/model/AggregateType.kt

```diff
@@ -1,10 +1,10 @@
-package com.kinvey.java.model;
+package com.kinvey.java.model
 
 /**
  * Created by yuliya on 10/12/17.
  */
 
-public enum AggregateType {
```

Comment:
I'm thinking if we should turn this private, if so it would be a breaking change

Reply:
By default, if no access modifier placed for `Kotlin` class it uses public access.